resources/definitions/fdmprinter.def.json

```diff
@@ -816,6 +816,15 @@
                     "minimum_value_warning": "0",
                     "maximum_value_warning": "machine_nozzle_size",
                     "settable_per_mesh": true
+                },"outer_inset_first": 
+                {
+                    "label": "Print Outer Inset First",
```

Comment:
Change to `Outer Before Inner Walls`.
In the Cura frontend the insets are always called 'walls'.